Detections/LAQueryLogs/UserSearchingForVIPUserActivity.yaml

```diff
@@ -20,8 +20,10 @@ relevantTechniques:
 query: |
   // Replace these with the username or emails of your VIP users you wish to monitor for.
   let vips = dynamic(['vip1@email.com','vip2@email.com']);
+  // Add users who are allowed to conduct these searches - this could be specific SOC team members
+  let allowed_users = dynamic([]);
   LAQueryLogs
-  | where QueryText has_any (vips)
+  | where QueryText has_any (vips) or QueryText has_any ('_GetWatchlist("VIPUsers")', "_GetWatchlist('VIPUsers')")
   | project TimeGenerated, AADEmail, RequestClientApp, QueryText, ResponseRowCount, RequestTarget
```

Comment:
RequestTarget is an Azure Resource, can we add this to the entity mapping - https://docs.microsoft.com/en-us/azure/sentinel/entities-reference#azure-resource.  Also, RequestClientApp might fit in Name under [Cloud Application](https://docs.microsoft.com/en-us/azure/sentinel/entities-reference#cloud-application)

Reply:
Done